.github/workflows/provider-couchbase.yml

```diff
@@ -0,0 +1,150 @@
+name: provider-couchbase
+
+on:
+  push:
+    tags:
+      - 'provider-couchbase-v[0-9].[0-9]+.[0-9]+'
+  pull_request:
+    paths:
+      - '.github/workflows/provider-couchbase.yml'
+      - 'providers/couchbase/**'
+
+env:
+  working_directory: providers/couchbase
+permissions:
+  contents: read
+  packages: write
```

Comment:
Would you mind moving this `packages: write` from the top level to under the `permissions:` block under `build:` step (line 20)?

Reply:
Yep yep, I thought that I had to have the permission up here too but I think I'm mistaken